src/util-affinity.c

```diff
@@ -319,7 +319,38 @@ uint16_t UtilAffinityGetAffinedCPUNum(ThreadsAffinityType *taf)
     return ncpu;
 }
 
-#ifdef HAVE_DPDK
+#ifdef __linux__
+#ifdef UNITTESTS
+/**
+ * \brief Verify that the CPU requirement is met
+ * \retval 0 if the requirement is met, 2 if the test is skipped, negative number otherwise
+ */
+int UnitTestsUtilAffinityVerifyCPURequirement(void)
```

Comment:
this seems very much dpdk test specific, should it go into runmode-dpdk.c as a static helper func?